README.md

```diff
@@ -23,6 +23,9 @@ year={2019}
 To install the dependencies use `pip3 install -r requirements-cpu.txt -f https://download.pytorch.org/whl/torch_stable.html` or `pip3 install -r requirements-gpu.txt`.
 
 ## Getting started
+### CIFAR-FS
```

Comment:
Why don't you download it in a bash script like it is implemented for the other datasets?

Reply:
downloading from google drive is kind of a hustle...

Reply:
Did you upload the dataset?

Reply:
Yes

Reply:
Does the copyright of the data allow this? If yes, just upload it to somewhere where we can do an easy `wget`, for example another git repository